src/lib/x509/certstor_system_linux/certstor_linux.cpp

```diff
@@ -0,0 +1,18 @@
+/*
+* Certificate Store
+* (C) 1999-2019 Jack Lloyd
+* (C) 2019      Patrick Schmidt
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/build.h>
+
+#include <botan/certstor_linux.h>
+
+namespace Botan {
```

Comment:
Remove this module entirely, instead `Flatfile_Certificate_Store` default constructor handles everything